ui/pullcounter/pullcounter.js

```diff
@@ -12,8 +12,11 @@ class PullCounter {
     this.zone = null;
     this.bossStarted = false;
 
-    let cmd = JSON.stringify({ getSaveData: '' });
-    OverlayPluginApi.overlayMessage(OverlayPluginApi.overlayName, cmd);
+    callOverlayHandler({
+      call: 'cactbotLoadData',
```

Comment:
Makes sense. Not sure *how* readable it currently is since I think the stringified JSON is just stored as a string inside the JSON config file which could mean a lot of escaping characters. Should be easy enough to find and edit though if you only want to edit a number and know what you're looking for.

Reply:
Haha, fair.  Maybe readable is not the right word here, but it's more about storing the state with the rest of the config rather than having it be off in wherever local storage goes.